src/keycloak/index.js

```diff
@@ -11,23 +11,9 @@ let initOptions = {
 keycloak.init(initOptions).then((auth) => {
 
     if (!auth) {
-        throw "Some error";
+        throw "Auth error";
     }
 
-    setInterval(() => {
-        keycloak.updateToken(70).then((refreshed) => {
-            if (refreshed) {
-                console.log('Token refreshed');
-            } else {
-                console.log('Token not refreshed, valid for '
-                    + Math.round(keycloak.tokenParsed.exp + keycloak.timeSkew - new Date().getTime() / 1000) + ' seconds');
-            }
-        }).error(() => {
-            console.log('Failed to refresh token');
-        });
-
-    }, 60000)
-

```

Comment:
I delete this because we don't want to automatically refresh the tokens; we only refresh them when the user is actively clicking button that result in API requests.